pkg/cli-runtime/printer/tabwriter/tabwriter.go

```diff
@@ -319,6 +322,10 @@ func (b *Writer) writeLines(pos0 int, line0, line1 int) (pos int) {
 		// if TabIndent is set, use tabs to pad leading empty cells
 		useTabs := b.flags&TabIndent != 0
 
+		if len(line) > 0 && b.paddingStart {
```

Comment:
What is the padding value here? How many spaces padding is done by default and should this be configurable from TabWriter struct?

Reply:
I'm using the default spaces that are used in the whole table. But we can definitely use another variable for the initial spaces for indentation

Reply:
[This line](https://github.com/vmware-tanzu/apps-cli-plugin/pull/247/files#diff-cbbb1be8f81b9b35121a45c800ea1decc899ba327a0e8fc36f853808fe3e4164L31) is the one that sets the number of padding spaces at the beginning of the table